shared/video-sdk/_authentication-workflow.mdx

```diff
@@ -33,14 +33,14 @@ When users attempt to connect to an <Vg k="COMPANY" /> channel from your <Vpl k=
 
 The following figure shows the call flow you need to implement to create step-up-authentication with <Vg k="COMPANY" /> <Vpd k="NAME"/>:
 
-![token authentication flow](/images/video-sdk/authentication-logic.png)
+![token authentication flow](/images/video-sdk/authentication-logic.svg)
```

Comment:
This is the diagram:
![image](https://user-images.githubusercontent.com/42718091/199661252-2d5fe407-e29b-4449-8c49-9341c4c48c18.png)